poll/poll.py

```diff
@@ -232,12 +234,12 @@ def get_max_submissions(data, result, private_results):
         except (ValueError, KeyError):
             max_submissions = 1
             result['success'] = False
-            result['errors'].append('Maximum Submissions missing or not an integer.')
+            result['errors'].append(_('Maximum Submissions missing or not an integer.'))
```

Comment:
@asadiqbal08 I think this should be translated dynamically using `self.ugettext`; `_` only marks it for translation.